packages/geo-tools/package.json

```diff
@@ -46,6 +47,7 @@
     "@bentley/imodeljs-frontend": "^2.9.1",
     "@bentley/imodeljs-i18n": "^2.9.1",
     "@bentley/itwin-client": "^2.9.1",
+    "@bentley/react-scripts": "3.4.6",
```

Comment:
any reason for adding this? i assume you were adding this to pickup the sprite-loader, but looks like that a dependency has also been added

Reply:
"@bentley/react-scripts" is referenced is tsconfig.json.  Without it, its impossible to import SVG files.

Reply:
I believe what you really need from this pkg is the following type declaration. 

```
declare module '*.svg?sprite' {
  const src: string;
  export default src;
}
```

I recommend you create a `geotools/declaration.d.ts` file with the above snippet, and remove the dep on react-scripts. Could you try that and let me know if that works?